dist/drawer-dialog/drawer-dialog.css

```diff
@@ -27,7 +27,7 @@
 .drawer-dialog__header {
     display: flex;
     flex-shrink: 0;
-    margin: var(--spacing-200) 0;
+    margin: --var(--spacing-250) var(--spacing-200) 0;
```

Comment:
This looks broken, but weren't we going to officially deprecate `drawer`? I'd vote to deprecate, but if we keep it around, we should fix the spacing. 
<img width="1688" alt="image" src="https://github.com/eBay/skin/assets/1675667/d4a8a2ef-dc07-4074-8b4f-8d4d1e8e5015">

Reply:
This was a bug when I did the sass change. This change should fix that.